src/main/java/com/selfxdsd/todocli/TodoParser.java

```diff
@@ -92,7 +92,8 @@ public TodoParser() {
 
             final String[] parts = line.substring(headerIndex)
                 .trim().split("\\s", 2);
-            if (parts[0].isEmpty()) {
+            if (parts[0].isEmpty() || !parts[0]
+                .matches("#\\d+:\\d+(min|days)")){
```

Comment:
@amihaiemil Oh, I forgot to ask:
This is matching against `min` and `days` (to pass one test that has days in it), but what about hours?
And what are the accepted forms for minutes/hours/days? like: 
- minutes: `m`,`min`,`mins`
- hours: `h`, `hours`
- days: `d`, `days`

And one more thing:
https://github.com/self-xdsd/todo-finder-cli/blob/9cbcc7071b804a4bb34dc6d0ef0c430115d30bad/src/main/java/com/selfxdsd/todocli/TodoParser.java#L170
When setting the estimation in `Todo`, it assumes that estimation time is in minutes so is no conversion if estimation is in days for example.

Reply:
@criske right. Regex for days/hours/whatever should be removed, I think. We should only work with minutes :D 

We can work with m, min and mins, if it's not complicated. If it's complicated, then just min should be enough.
Please open a ticket for this :)